Sources/SwerverCore/Router/Routes.swift

```diff
@@ -2,19 +2,19 @@ import Foundation
 
 public class Routes {
 
-    private var allRoutes: [URL:[RequestMethod:HttpAction]] = [:]
+    private var allRoutes: [URL:Route] = [:]
```

Comment:
I really like the introduction of a `Route` abstraction.

Reply:
😺